lib/ansible/modules/systemd.py

```diff
@@ -323,9 +325,33 @@ def parse_systemctl_show(lines):
     return parsed
 
 
+def enable_linger(module):
+    ''' allows for lingering dbus sessions '''
+
+    enabled = False
+    user = getpass.getuser()
```

Comment:
This may not always be the correct user account and may differ from the value in `XDG_RUNTIME_DIR`.